src/components/TutorialCarouselDialog/styles.js

```diff
@@ -58,23 +65,52 @@ const styles = (theme) => ({
     marginBottom: theme.padding.xs.px,
   },
   contentWrapper: {
-    padding: `${theme.padding.sm.px} ${theme.padding.md.px} ${theme.padding.md.px} ${theme.padding.md.px} !important`,
+    padding: `${theme.padding.sm.px} ${theme.padding.md.px} ${theme.padding.sm.px} ${theme.padding.md.px}`,
+    [theme.breakpoints.down('xs')]: {
+      padding: `${theme.padding.unit.px} ${theme.padding.sm.px} 0 ${theme.padding.sm.px}`,
+    },
+  },
+  tutorialDialogContentWrapper: {
+    height: 400,
+    overflowX: 'auto',
```

Comment:
remove, everything should collapse so we can view all the content with Y scrolling

Reply:
![image](https://user-images.githubusercontent.com/18548885/46063466-77e11280-c19f-11e8-8647-be9f56f0202d.png)
![image](https://user-images.githubusercontent.com/18548885/46063516-a65eed80-c19f-11e8-9366-cb4351322d5f.png)
The user needs to scroll to touch buttons. Do you mean this?

Reply:
Nope, dont want the user to scroll to see the buttons. Just want to get rid of the horizontal scroll bar.